File: test/utils/Config/dbConfig.js
require('dotenv').config({ path: '../../.env' });

const dbConfig = {
  host: process.env.DB_HOST || '',
  user: process.env.DB_USER || '',
  password: process.env.DB_PASSWORD || '',
  database: process.env.DB_NAME || '',
};

module.exports = dbConfig;

File: utils/config/dbconfig.py
import os
from dotenv import load_dotenv

load_dotenv("../../.env")

db_config = {
    "host": os.getenv("db_config.host"),
    "port": os.getenv("db_config.port"),
    "user": os.getenv("db_config.user"),
    "password": os.getenv("db_config.password"),
    "database": os.getenv("db_config.database")
}